week1/assignment/웨비의 사진관 📸/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css" />
+    <link rel="shortcut icon" href="./img/favicon.ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>수빈 | 삿포로 여행 계획</title>
+</head>
+
+<body>
+    <header>
+        <h1>🍣 🍶 🧤 🍡 &nbsp;삿포로 여행 계획&nbsp; 🍣 🍶 🧤 🍡</h1>
+    </header>
+    <section id="preview">
+        <img src="./img/biei.jpg" alt="비에이" />
+        <img src="./img/vending.JPG" alt="자판기" />
+        <img src="./img/tv-tower.png" alt="tv타워" />
+        <img src="./img/nikka.png" alt="니카상" />
+        <img src="./img/ryokan2.jpg" alt="료칸" />
+        <img src="./img/otaru-canal.jpg" alt="오타루운하" />
+        <img src="./img/ningle-terrace.jpg" alt="닝구르테라스" />
+        <img src="./img/ski.jpg" alt="스키장" />
+        <img src="./img/sarutama.jpg" alt="사루타마온천" />
+    </section>
+    <nav>
+        <p>클릭하면 이동해요 👉🏻</p>
+        <ul>
+            <a href="#day1">
+                <li>DAY 1️⃣</li>
+            </a>
+            <a href="#day2">
+                <li>DAY 2️⃣</li>
+            </a>
+            <a href="#day3">
+                <li>DAY 3️⃣</li>
+            </a>
+        </ul>
```

Comment:
```suggestion
 <ul>
            <li >
                <a href="#day1">DAY 1️⃣</a>
            </li >
            <li >
                <a href="#day2">DAY 2️⃣</a>
            </li>
            <li >
                <a href="#day3">DAY 3️⃣</a>
            </li>
        </ul>
```
이렇게 묶어주어야 각 a 태그가 각각의 list에 속하는구나~! 의 구조가 되지 않을까용?